amplify/backend/api/vocelapp/cli-inputs.json

```diff
@@ -0,0 +1,23 @@
+{
+  "version": 1,
+  "serviceConfiguration": {
+    "apiName": "vocelapp",
+    "serviceName": "AppSync",
+    "defaultAuthType": {
+      "mode": "AMAZON_COGNITO_USER_POOLS",
+      "cognitoUserPoolId": "authvocelappcf1453cb"
```

Comment:
Is this something that someone running in their own backend would need to change?

Reply:
YES, they will use their own user pool!